### ISSUE TITLE
borked dockerfile

### ISSUE THREAD

**Issue by zenfish** [skavngr/rapidscan#34]
MacOS 10.15.7 (Catalina)

TLDR; run an apt-get before anything else... you either need to ignore keys altogether or use some alternate method to get the Kali pubkey (curl, wget, etc. aren't in the baseimage.)  Alternate workaround after errors building:

`Sending build context to Docker daemon  3.565MB
Step 1/10 : FROM kalilinux/kali-rolling
 ---> c2fadbc65f8d
Step 2/10 : ENV DEBIAN_FRONTEND noninteractive
 ---> Using cache
 ---> fd58a73e4d8b
Step 3/10 : RUN apt-get update && apt-get -yu dist-upgrade -y
 ---> Running in 73b26ccf950e
Get:1 http://mirrors.ocf.berkeley.edu/kali kali-rolling InRelease [30.6 kB]
Err:1 http://mirrors.ocf.berkeley.edu/kali kali-rolling InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY ED444FF07D8D0BF6
Reading package lists...
W: http://http.kali.org/kali/dists/kali-rolling/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/debian-archive-bullseye-automatic.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://http.kali.org/kali/dists/kali-rolling/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/debian-archive-bullseye-security-automatic.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://http.kali.org/kali/dists/kali-rolling/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/debian-archive-bullseye-stable.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://http.kali.org/kali/dists/kali-rolling/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/debian-archive-buster-automatic.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://http.kali.org/kali/dists/kali-rolling/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/debian-archive-buster-security-automatic.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://http.kali.org/kali/dists/kali-rolling/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/debian-archive-buster-stable.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://http.kali.org/kali/dists/kali-rolling/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/debian-archive-stretch-automatic.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://http.kali.org/kali/dists/kali-rolling/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/debian-archive-stretch-security-automatic.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://http.kali.org/kali/dists/kali-rolling/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/debian-archive-stretch-stable.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: http://http.kali.org/kali/dists/kali-rolling/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/kali-archive-keyring.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: GPG error: http://mirrors.ocf.berkeley.edu/kali kali-rolling InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY ED444FF07D8D0BF6
E: The repository 'http://http.kali.org/kali kali-rolling InRelease' is not signed.
The command '/bin/sh -c apt-get update && apt-get -yu dist-upgrade -y' returned a non-zero code: 100`

You can do the somewhat crazy :

`
RUN apt -o Acquire::AllowInsecureRepositories=true -o Acquire::AllowDowngradeToInsecureRepositories=true update
`

Or actually get the key somehow.  If you do the above, three more changes were necessary to make the dockerfile work -

1) add the "--allow-unauthenticated" to `apt-get` RUN line
2) change python to python3 in the `apt-get` package listing.
3) change the last line from:  `ENTRYPOINT ["/usr/local/bin/rapidscan.py"]` -> `ENTRYPOINT ["python3", "/usr/local/bin/rapidscan.py"]`

**Comment by tristanlatr** [comment]
I have submitted a PR that fixes the Dockerfile, #33. But in my PR, I do not change anything regarding point 1, and it works as expected. Please try the Dockerfile at https://github.com/tristanlatr/rapidscan/tree/fix-docker-python3 and give me some feedback. Iw works on my computer.

**Comment by sh4nx0r** [comment]
I have merged the PR. @tristanlatr @zenfish
